models/engine/db_storage.py

```diff
@@ -74,3 +74,27 @@ def reload(self):
     def close(self):
         """call remove() method on the private session attribute"""
         self.__session.remove()
+
+    def get(self, cls, id):
+        """
```

Comment:
Make doctrings like Google Style, ref: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html
In this, the description is first, after arguments and next return